doc/monitoring/metrics_reference.rst

```diff
@@ -378,7 +378,7 @@ efficient.
     above 5% indicates that vinyl is not healthy. At this moment you'll probably
     see a lot of other problems with vinyl.
 
-*   ``tnt_vinyl_tx_read_views``—the counter of read views, that is, transactions
+*   ``tnt_vinyl_tx_read_views``—the amount of read views, that is, transactions
```

Comment:
I suggest to add metric type - this is gauge. Also I think this is "number" not amount.